### PR TITLE
Fillers and Reward Points indexing

### PR DESCRIPTION
**Pull request by omadoyeabraham**
- Added indexing for Bids Placed and Bids Refunded
- Added indexing for Requests Fulfilled
- Added Indexing for Assets received and assets teleported
- Implemented Rewards system assigning reward points to USERS, FILLERS and RELAYERS depending on transaction statuses and the dollar values of assets handled

### CODE REVIEW

File: indexers/schema.graphql [hunk 1]
```diff
@@ -15,6 +15,17 @@ enum SupportedChain {
...
 
+"""
+Assets currently supported by hyperbridge
+"""
+enum SupportedAssets {
+    WETH
+    WBTC
+    USDT
+    USDC
+    DAI
```
Comment (seunlanlege) [inline indexers/schema.graphql:26]: lets add DOT as well

File: indexers/schema.graphql [hunk 2]
```diff
@@ -364,3 +392,85 @@ type ResponseStatusMetadata @entity {
...
+    id: ID!
+    address: String!
+    chain: SupportedChain!
+    points: BigInt!
+    earnerType: ProtocolParticipant!
+}
+
+type RewardPointsActivityLog {
+    id: ID!
+    chain: SupportedChain!
```
Comment (seunlanlege) [inline indexers/schema.graphql:470]: feel as though we should add transaction hash?

File: indexers/schema.graphql [hunk 3]
```diff
@@ -38,6 +49,23 @@ enum Status {
...
+    FILLER
+    USER
+}
+
+"""
+System activity types related to reward points
+"""
+enum RewardPointsActivityType {
+    REWARD_POINTS_EARNED
+    REWARD_POINTS_SLASHED
```
Comment (Wizdave97) [inline indexers/schema.graphql:66]: There's no slashing
```suggestion
```

File: indexers/schema.graphql (hunk 1 above)
Comment (omadoyeabraham) [reply indexers/schema.graphql:26]: DOT addes

File: indexers/schema.graphql (hunk 2 above)
Comment (omadoyeabraham) [reply indexers/schema.graphql:470]: Done

File: indexers/schema.graphql (hunk 3 above)
Comment (omadoyeabraham) [reply indexers/schema.graphql]: Removed

**Comment by Wizdave97** [comment]
@omadoyeabraham  pull. changes from main and run cargo fmt with the same version of rust used in the CI

**Review by Wizdave97 — APPROVED** [review]

**Pull request merged**